src/session/mod.rs

```diff
@@ -367,6 +367,15 @@ impl Session {
         }
     }
 
+    pub(crate) fn cancel_download_file(&self, file_id: i32) {
+        let client_id = self.client_id();
+        spawn(async move {
+            if let Err(e) = functions::cancel_download_file(file_id, false, client_id).await {
+                log::warn!("Error canceling a file: {:?}", e);
```

Comment:
```suggestion
                log::warn!("Error canceling a file: {e:?}");
```

Reply:
I took the code from line 363
![image](https://user-images.githubusercontent.com/63008755/194752563-485b1106-cc68-4a72-b29b-006610525abb.png)
I'm not sure that inline variant is better because it's not highlithed correctly

Also all other code uses this style
![image](https://user-images.githubusercontent.com/63008755/194752739-24d952b0-30ed-4979-a6b9-cbf300512ade.png)

Reply:
Yeah it's not used consistently, we've started using it lately. You can keep it for now and then we can make it more consistent across the entire project later.

Reply:
Ok